language/tools/move-package/src/resolution/resolution_graph.rs

```diff
@@ -515,6 +515,30 @@ impl ResolvingGraph {
         }
     }
 
+    pub fn download_dependency_repos(
+        manifest: &SourceManifest,
+        build_options: &BuildConfig,
+        root_path: &Path,
+    ) -> Result<()> {
+        // include dev dependencies if in dev mode
+        let additional_deps = if build_options.dev_mode {
+            manifest.dev_dependencies.clone()
```

Comment:
This clone isn't needed if you are just using `.iter()` later.

I imagine you did this `clone` due to Rust borrow checker issues, and it is indeed an annoying one. 

You might need to do this:
```
let empty_deps;
let additional_deps = if build_options.dev_mode {
        manifest.dev_dependencies.clone()
} else {
    empty_deps = BTreeMap::new();
    &empty_deps
};
```